### PR TITLE
WPF support for ARM64: Enable build in public repo and update packaging

### PR DESCRIPTION
**Pull request by ryalanms**
Complete any remaining packaging and pipeline updates and enable ARM64 builds in the WPF repo. 

Requires #4034, #4035, #4041, and #4042 to be completed beforehand.

Co-authored-by: Alexandre Zollinger Chohfi <...>
Co-authored-by: Marcpems <...>
Co-authored-by: Juan Sebastian Hoyos Ayala <...>
Co-authored-by: Ryland <41491307+ryalanms@users.noreply.github.com>

/cc @azchohfi @hoyosjs @marcpems

### CODE REVIEW

**Comment by hoyosjs** [comment]
Any reason not to include the packaging + pipeline changes here?

**Comment by ryalanms** [comment]
We could combine them, I suppose.

File: NuGet.config [hunk 1]
```diff
@@ -9,7 +9,6 @@
     <add key="dotnet5-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5-transport/nuget/v3/index.json" />
     <add key="dotnet6" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json" />
     <add key="dotnet6-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6-transport/nuget/v3/index.json" />
-    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```
Comment (hoyosjs) [inline NuGet.config:12]: This looks like an unintentional change.

**Review by hoyosjs — APPROVED** [review]

**Comment by ryalanms** [comment]
This PR build will not pass until the other dependent PRs are merged.

**Review by azchohfi — CHANGES_REQUESTED** [review]
Only the nuget.config change seems wrong.

**Review by azchohfi — APPROVED** [review]

**Comment by ryalanms** [comment]
There are failures in our APICompat checks.  Investigating...

**Pull request merged**